rules/windows/lateral_movement_executable_tool_transfer_smb.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/11/10"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or change of a Windows executable file over network shares. Adversaries may transfer tools or
+other files between systems in a compromised environment.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Lateral Tool Transfer"
+risk_score = 47
+rule_id = "58bc134c-e8d2-4291-a552-b4b3e537c60b"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence with maxspan=5s
+  [network where event.type == "start" and process.pid == 4 and destination.port == 445 and
+   event.action == "connection_accepted" and network.direction == "incoming" and network.transport == "tcp" and
+   source.address != "127.0.0.1" and source.address != "::1"
+  ] by host.id, process.entity_id
+  /* add more executable extensions here if they are not noisy in your environment */
+  [file where event.type in ("creation", "change") and process.pid == 4 and file.extension in ("exe", "dll", "bat")] by host.id, process.entity_id
+'''
```

sick rule. i love this idea. one thing to note -- i believe that the 445 outgoing connection will happen when the share is mounted, and sometimes the connection is kept alive for a transfer which could happen later. like if the `net use` is separate from a `cmd /c copy`, you might FN because the window is too small. I think you can probably get away with a 30s window, maybe larger.


since `host.id` is common, you can move this to the top. also I increased `maxspan` since I think you'll be okay
```suggestion
sequence by host.id with maxspan=10s
  [network where event.type == "start" and process.pid == 4 and destination.port == 445 and
   event.action == "connection_accepted" and network.direction == "incoming" and network.transport == "tcp" and
   source.address != "127.0.0.1" and source.address != "::1"
  ] by =process.entity_id
  /* add more executable extensions here if they are not noisy in your environment */
  [file where event.type in ("creation", "change") and process.pid == 4 and file.extension in ("exe", "dll", "bat")] by process.entity_id
'''
```